### PR TITLE
feature/gradels

### PR DESCRIPTION
**Pull request by MoatazBadawy**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -40,78 +47,68 @@ android {
...
     implementation 'com.google.firebase:firebase-analytics-ktx'
     implementation 'com.google.firebase:firebase-crashlytics'
     // firebase message
-    implementation 'com.google.firebase:firebase-messaging:23.0.7'
-    // Navigation component
-    implementation 'androidx.navigation:navigation-fragment-ktx:2.5.1'
-    implementation 'androidx.navigation:navigation-ui-ktx:2.5.1'
-    // swipe to refresh
-    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
+    implementation 'com.google.firebase:firebase-messaging:23.1.1'
```
Comment (MahmoudMabrok) [inline app/build.gradle:84]: why not use the one comes with `BOM`

**Review by MahmoudMabrok — COMMENTED** [review]
good work, we should plan to use gradle kotlin dsl

File: app/build.gradle (hunk 1 above)
Comment (MoatazBadawy) [reply app/build.gradle:84]: I didn't notice it, ok I'll use it.

**Comment by MoatazBadawy** [comment]
Thank you :), I'll put that in the eye of recognition.

**Pull request merged**